src/main/java/ladder/domain/player/Players.java

```diff
@@ -0,0 +1,49 @@
+package ladder.domain.player;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import ladder.exception.ErrorMessage;
+import ladder.exception.InvalidInputException;
+
+public class Players {
+    public static final int MAXIMUM_PLAYER_SIZE = 10;
+    public static final int MINIMUM_PLAYER_SIZE = 2;
+    private final List<Player> players;
+
+    public Players(final List<Player> players) {
+        validate(players);
+        this.players = players;
+    }
+
+    private void validate(final List<Player> players) {
+        validateSize(players);
+        validateDuplicatedName(players);
+    }
+
+    private void validateSize(final List<Player> players) {
+        final int playerSize = players.size();
+
+        if (playerSize > MAXIMUM_PLAYER_SIZE || playerSize < MINIMUM_PLAYER_SIZE) {
```

Comment:
가독성 초미세팁인데요, 비교 인자를 가운데로 모아주거나 바깥쪽으로 빼주는게 가독성이 조금더 좋은거 같습니다
```suggestion
        if (MINIMUM_PLAYER_SIZE < playerSize || playerSize < MAXIMUM_PLAYER_SIZE) {
```

Reply:
그렇군요. 꿀팁 감사합니다.
수정했습니다!